.github/workflows/deployment_test.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         uses: SamKirkland/FTP-Deploy-Action@v4.3.5
         with:
           server: files.dice-research.org
```

Comment:
it should be ftp over tls. Not sure how to set that up here

Reply:
Yeah I thought so too, but something didn't work on the first run

Reply:
maybe `ftps-legacy`?

Reply:
didn't work either

Reply:
I don't know if this is relevant but it seems to be a common [issue](https://github.com/SamKirkland/FTP-Deploy-Action/issues/106) . I think I will just try to use lftp itself.